routes/api/accounts.js

```diff
@@ -0,0 +1,53 @@
+const express = require( 'express' )
+
```

Comment:
Please group node_module `require`s together, followed by `require`s of local modules, followed by constant declarations, with each block separated by whitespace.